src/clj/game/cards/events.clj

```diff
@@ -69,6 +69,10 @@
     :effect (effect (resolve-ability {:prompt "Choose a server" :choices (req runnable-servers)
                                       :effect (effect (run target nil card))} card nil))}
 
+   "Build Script"
+   {:msg (msg "gain 1 [Credits] and draw 2 cards")
```

Comment:
`(msg .... )` is only required if you are using any functions to determine the contents of the message.